coderd/provisionerdserver/acquirer.go

```diff
@@ -476,7 +480,7 @@ func (d domain) contains(p provisionerjobs.JobPosting) bool {
 }
 
 func (d domain) poll(dur time.Duration) {
-	tkr := time.NewTicker(dur)
+	tkr := d.clock.NewTicker(dur, "poll", string(d.key))
```

Comment:
I usually recommend `TickerFunc` for this sort of thing, since you can then use Quartz to wait for `clearOrPend` to complete.